src/app/loadout-builder/generated-sets/GeneratedSet.tsx

```diff
@@ -90,11 +91,32 @@ function GeneratedSet({
     ? assignModsToArmorSet(set.firstValidSet, lockedArmor2Mods)
     : {};
 
+  const incorrectStats = _.uniq(
```

Comment:
Might be nice to extract a helper for this?

Reply:
I will do the stat display component you mentioned in the top comment and pull this out as part of that.